srcpkgs/Aegisub/files/wx-cross.ini

```diff
@@ -0,0 +1,2 @@
+[binaries]
+wx-config = /usr/bin/wx-config-gtk3
```

Comment:
This is the wrong path. You should be running the xbps-src wrapper. Also, just write this file dynamically from a heredoc in `pre_configure` rather than storing the file in the source hierarchy.